service/__main__.py

```diff
@@ -1,12 +1,18 @@
 import os
 import asyncio
 import base64
+import importlib
 
 from aioimaplib import aioimaplib
 import mailparser
 import requests
 from requests_toolbelt import MultipartEncoder
 
+dotenv_spec = importlib.util.find_spec("dotenv")
+if dotenv_spec is not None:
+    from dotenv import load_dotenv
+    load_dotenv()
```

Comment:
i'm more used to doing a try catch ImportError, I think it's more pythonic (ask for forgiveness, not permission)